docs/source/md/cli_references.md

```diff
@@ -0,0 +1,63 @@
+## Command Line Interface
+
+Aim CLI offers a simple interface to easily organize and record your experiments.
+Paired with the [Python Library](#sdk-specifications), Aim is a powerful utility to record, search and compare AI experiments.
+Here are the set of commands supported:
+
+
+| Command       | Description                                                          |
+| --------------| -------------------------------------------------------------------- |
+| `init`        | Initialize the `aim` repository.                                     |
+| `version`     | Displays the version of aim cli currently installed.                 |
+| `experiment`  | Creates a new experiment to group similar training runs into.        |
```

Comment:
`aim experiment` command had been removed.

Reply:
also need to add `aim upgrade` command here. It's used to convert legacy repos to the v3.0.0 format (and will be used for future backwards-incompatible changes as well)